profiler/data.py

```diff
@@ -0,0 +1,137 @@
+import os
+import pickle
+import re
+from datetime import date, datetime
+from typing import Optional
+
+import somacore
+
+import tiledbsoma
+
+
+class ProfileData:
+    """This class represent the dat stored per run"""
+
+    date = str(date.today())
+    now = str(datetime.now())
+    idx = now.find(".")
+    if idx != -1:
+        now = now[: idx - 1]
```

Comment:
all of these vars should be set in __init__, otherwise they are class variables and won't change for new instances of ProfileData. This is just a latent bug for now, but worth fixing.

Might also have the caller just pass in the timestamp, to make this "struct"-like class a bit dumber. :)

Reply:
+1 for Andrew's suggestion. Also consider using `datetime.utcnow()` to avoid potential issues with timezones.

Reply:
Thanks. These all make sense.